NEWS

```diff
@@ -11,6 +11,11 @@
 ### Features:
 ### Bugfixes:
 
+## 1.15.0-rc5 (September 10, 2023)
+### Bugfixes:
+#### UCP
+Fixed the data race on endpoint configurations.
```

Comment:
Please add `*` in the beginning of the line

Reply:
Fixed.
Force-pushed the fix.